recipes/interpret-community/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set name = "interpret-community" %}
+{% set version = "0.29.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/interpret_community-{{ version }}.tar.gz
+  sha256: 48e93f34a6563c3a0a9718d4a2e844e2cc273eabde437bdb40b9d8ff182514c9
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - numpy
+    - pandas <2.0.0
+    - scipy
+    - ml-wrappers >=0.4.3,<0.5.dev0
+    - scikit-learn
+    - packaging
+    - interpret-core >=0.1.20,<=0.3.2
+    - shap >=0.20.0,<=0.41.0
+    - raiutils >=0.4.0,<0.5.dev0
+
+test:
+  imports:
+    - interpret_community
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/interpretml/interpret-community
+  summary: Microsoft Interpret Extensions SDK for Python
+  license: MIT
+  license_file: LICENSE.txt
+  description: |
+    **Microsoft Interpret Community SDK for Python**: `{{ name | lower }}`
+
+    The Interpret Community SDK builds on Interpret, an open source python package
+    from Microsoft Research for training interpretable models, and helps to explain
+    blackbox systems by adding additional extensions from the community to interpret
+    ML models.
+
+    - `{{ name | lower }}`: Interpret-Community is an experimental repository that hosts
+    a wide range of community developed machine learning interpretability techniques.
+    This repository makes it easy for anyone involved in the development of a machine
+    learning system to improve transparency around their machine learning models. Data
+    scientists, machine learning engineers, and researchers can easily add their own
+    interpretability techniques via the set of extension hooks built into the peer
+    repository, Interpret, and expand this repository to include their custom-made
+    interpretability techniques.
+
+    PyPI: [https://pypi.org/project/{{ name | lower }}/](https://pypi.org/project/{{ name | lower }}/)
+
+    ---
+
+    > The conda-forge recipe was made with [CondaForger App](https://sugatoray-conda-forger.streamlit.app/).
```

Comment:
This description is quite long and some parts of it doesn't seem to be necessary for the recipe info. Consider reducing it a bit.